### PR TITLE
Add accessibility section for `Stat`

### LINKED ISSUE #1573: Add accessibility section for `stat`
### Subject

accessibility section for `stat`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )

#### Include the following content
* aria, tabindex property is supported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by setodeve**
Closes #1573

## Description
Added accessibility documentation for `Stat`.

## Is this a breaking change (Yes/No):

No

## Additional Information
None

### CODE REVIEW

**Review by 108yen — APPROVED** [review]

**Review by suzukisan22 — DISMISSED** [review]

**Comment by setodeve** [comment]
i'll add `order`

File: docs/contents/components/data-display/stat/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性 | 使い方                                                                                                                 |
+| -------------- | ------------ | ---------------------------------------------------------------------------------------------------------------------- |
+| `Stat`         | `icon`       | `"increase"`または`"decrease"`が設定されている場合、`StatIcon`に`type`として渡されます。                               |
```
Comment (hirotomoyamada) [inline docs/contents/components/data-display/stat/accessibility.ja.mdx:14]: There is no `icon` role or attributes.This is unique to Yamada UI.Please remove this line.

File: docs/contents/components/data-display/stat/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+| `StatIcon`     | `type`       | `"increase"`または`"decrease"`が設定できます。                                                                         |
```
Comment (hirotomoyamada) [inline docs/contents/components/data-display/stat/accessibility.ja.mdx:15]: There is no `type` role or attributes.This is unique to Yamada UI.Please remove this line.

File: docs/contents/components/data-display/stat/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+## ARIA Roles and Attributes
+
+| Component  | Role and Attributes | Usage                                                                                                  |
+| ---------- | ------------------- | ------------------------------------------------------------------------------------------------------ |
+| `Stat`     | `icon`              | If set to `"increase"` or `"decrease"`, it is passed as `type` to `StatIcon`.                          |
```
Comment (hirotomoyamada) [inline docs/contents/components/data-display/stat/accessibility.mdx:14]: There is no `icon` role or attributes.This is unique to Yamada UI.Please remove this line.

File: docs/contents/components/data-display/stat/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+| `StatIcon` | `type`              | Can be set to `"increase"` or `"decrease"`.                                                            |
```
Comment (hirotomoyamada) [inline docs/contents/components/data-display/stat/accessibility.mdx:15]: There is no `type` role or attributes.This is unique to Yamada UI.Please remove this line.

File: docs/contents/components/data-display/stat/accessibility.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+|            | `aria-label`        | If `type` is set to `"increase"`, it will be `"Increased by"`. Otherwise, it will be `"Decreased by"`. |
```
Comment (hirotomoyamada) [inline docs/contents/components/data-display/stat/accessibility.mdx:16]: Change

```
`type`に`"increase"`が設定されている場合、`"Increased by"`が設定されます。それ以外は、`"Decreased by"`が設定されます。
```

to

```
`type`に`"increase"`が設定されている場合は`"Increased by"`を設定し、`"decrease"`が設定されている場合は`"Decreased by"`を設定します。
```

Please correct the English as well.

**Comment by setodeve** [comment]
@hirotomoyamada 
Could you please check it again?

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**